### ISSUE TITLE
Unable to open device

### ISSUE THREAD

**Issue by Qbicz** [apmorton/pyhidapi#19]
Hi, I'm trying to use pyhidapi to send feature reports. This is a device normally claimed by the Windows operating system (HID mouse).
I needed to apply that patch to the HIDAPI C library: https://github.com/signal11/hidapi/pull/335
With it, I can open the device with C code and send feature reports, however with `pyhidapi` I can enumerate devices, but trying to create a `hid.Device(vid, pid)` results in 
```
  File "C:\Python27\lib\site-packages\hid-0.1.1-py2.7.egg\hid\__init__.py", line 91, in __init__
    raise HIDException('unable to open device')
hid.HIDException: unable to open device
```
I use the DLL with the mentioned patch. Any ideas?

**Comment by apmorton** [comment]
The version on pypi is outdated - it doesn't include the fixes for correct operation on Windows, which I finally merged today. Can you try with the code from master and see if that works?

**Comment by apmorton** [comment]
Also, fwiw, prior to #12/#13 it will still locate and load hidapi.dll on Windows, it just uses the incorrect calling convention and nothing works.

**Comment by Qbicz** [comment]
I have Python2.7 [MSC v.1500 32 bit (Intel)] on win32 so I compiled the `hidapi.dll` with Visual Studio 2008. However, it didn't work with opening the device. When I compiled it with Visual Studio 2017, it works!

Since we are at it, there is no conversion of arrays from Python to C for calls like `write()` or `send_feature_report()`. Should I just import `ctypes` and do something like
```
arr = (ctypes.c_int * len(pyarr))(*pyarr)
```
